docs/src/callbacks/summary.md

```diff
@@ -1 +1,19 @@
 # Callbacks
+
+factory\_bot makes four callbacks available:
+
+* after(:build)   - called after a factory is built   (via `FactoryBot.build`, `FactoryBot.create`)
+* before(:create) - called before a factory is saved  (via `FactoryBot.create`)
+* after(:create)  - called after a factory is saved   (via `FactoryBot.create`)
+* after(:stub)    - called after a factory is stubbed (via `FactoryBot.build_stubbed`)
+
+Examples:
+
+```ruby
+# Define a factory that calls the generate_hashed_password method after it is built
+factory :user do
+  after(:build) { |user, evaluator| generate_hashed_password(user) }
```

Comment:
I'm not sure if this is the point in time you're interested in feedback about verbiage, but I've found the usage of `evaluator` to be vague and somewhat confusing.

Anecdotally, we've been naming this second block argument as `factory`, and that has felt more self-documenting.

Reply:
I'm quite open to verbiage!

My hesitation about `factory` is that we also have a method with the same name. How bad would it be to call it `definition` instead?

```ruby
factory :user do
  after(:build) { |user, definition| ... }
end
```

Reply:
After a heated discussion in #dev , I've recognized that the best name for this parameter is `context`. Going to make that change. Holler if you hate it!